specs/scrabble.spec.js

```diff
@@ -181,45 +181,55 @@ describe('Player', () => {
   });
 
   describe('hasWon', () => {
-    test.skip('returns false when score < 100', () => {
-
-
+    test('returns false when score < 100', () => {
+      const player = new Scrabble.Player('test player');
```

Comment:
It would be good to loop through until you reach 99, the edge case, and verify that `hasWon()` is `false`.